meteor/lib/__tests__/lib.test.ts

```diff
@@ -398,4 +402,79 @@ describe('lib/lib', () => {
 		expect(MyCollection.find({ rank: { $lte: 3 } }).fetch()).toHaveLength(4)
 
 	})
+	testInFiber('getRank', () => {
+
+		const objs: {_rank: number}[] = [
+			{ _rank: 0 },
+			{ _rank: 10 },
+			{ _rank: 20 },
+			{ _rank: 21 },
+			{ _rank: 22 },
+			{ _rank: 23 },
+		]
+
+		// First:
+		expect(getRank(null, objs[0])).toEqual(-0.5)
+		// Insert two:
+		expect(getRank(null, objs[0], 0, 2)).toEqual(-0.6666666666666667)
+		expect(getRank(null, objs[0], 1, 2)).toEqual(-0.33333333333333337)
+
+		// Center:
+		expect(getRank(objs[1], objs[2])).toEqual(15)
+		// Insert three:
+		expect(getRank(objs[1], objs[2], 0, 3)).toEqual(12.5)
+		expect(getRank(objs[1], objs[2], 1, 3)).toEqual(15)
+		expect(getRank(objs[1], objs[2], 2, 3)).toEqual(17.5)
+
+		// Last:
+		expect(getRank(objs[5], undefined)).toEqual(23.5)
+		// Insert three:
+		expect(getRank(objs[5], undefined, 0, 3)).toEqual(23.25)
+		expect(getRank(objs[5], undefined, 1, 3)).toEqual(23.5)
+		expect(getRank(objs[5], undefined, 2, 3)).toEqual(23.75)
+
+		// Insert in empty list
+		expect(getRank(undefined, undefined)).toEqual(0.5)
+
+		// Insert three:
+		expect(getRank(undefined, undefined, 0, 2)).toEqual(0.3333333333333333)
+		expect(getRank(undefined, undefined, 1, 2)).toEqual(0.6666666666666666)
+
+	})
+	testInFiber('partial', () => {
+		const o = {
+			a: 1,
+			b: 'asdf',
+			c: {
+				d: 1
+			},
+			e: null,
+			f: undefined
+		}
+		expect(partial(o)).toEqual(o) // The function only affects typings
+	})
+	testInFiber('partialExceptId', () => {
+		const o = {
+			_id: 'myId',
+			a: 1,
+			b: 'asdf',
+			c: {
+				d: 1
+			},
+			e: null,
+			f: undefined,
+		}
+		expect(partialExceptId(o)).toEqual(o) // The function only affects typings
+	})
+	testInFiber('formatDateTime', () => {
+		expect(formatDateTime(1578295344070)).toBe('2020-01-06 08:22:24')
```

Comment:
This test doesn't pass on CircleCI, perhaps it's a problem with the Timezone?

Reply:
True, I'll look into it :)

Reply:
Fixed now, I ended up having to skip the specific tests when running locally on Windows, as there seems to be a bug in how the timezone settings are applied in Node on Windows. ref: https://www.npmjs.com/package/set-tz#windows-caveat